apps/state/config/config.exs

```diff
@@ -259,14 +259,35 @@ config :state, :stops_on_route,
 config :state, :stops_on_route,
   stop_order_overrides: %{
     {"CR-Franklin", 0} => [
-      ["Norwood Central", "Windsor Gardens", "Plimptonville", "Walpole"],
```

Comment:
It looks like there's a stops on route test failing around Plimptonville 🤔 
https://github.com/mbta/api/actions/runs/9071064580/job/24924071203#step:9:1205

Reply:
Oh interesting...I just removed that because we've removed those human-readable `stop_id`s from production years ago... Maybe I'll just adjust the test to use the IRL `stop_id`s.